packages/mobile/src/recipients/RecipientItem.tsx

```diff
@@ -14,8 +14,13 @@ interface Props {
 
 class RecipientItem extends React.PureComponent<Props> {
   onPress = () => {
-    const recipient = this.props.recipient
-    recipient.address = recipient.address?.toLowerCase()
+    let recipient = this.props.recipient
+    if (recipient.address) {
```

Comment:
Instead of modifying the the address casing here in the view layer (which shouldn't concern itself with data details liek this), how about we move this normalization into the sagas?

Reply:
That's a good point

Reply:
I moved this to call `toLowerCase` on the address search string, which is the only one that's using an address directly here.